Doc/library/contextlib.rst

```diff
@@ -314,6 +314,10 @@ Functions and classes provided:
       ``suppress`` now supports suppressing exceptions raised as
       part of an :exc:`ExceptionGroup`.
 
+   .. versionchanged:: 3.12.1
```

Comment:
I'm not sure we need this level of detail in the doc, TBH. It might be good enough to update the 3.12 entry with this change.